### DISCUSSION TITLE
MineStat Timing Out Issue

### DISCUSSION THREAD

**Discussion by Archangel4148** [FragLand/minestat#190] [Q&A] [answered]
Hello! I'm relatively new when it comes to implementing APIs in general, but I have come across an issue that I am unsure what to do about. I have successfully implemented MineStat into a small Discord bot that I am learning to build, and it works for some servers, but when I attempt to find the status of a larger server like mc.Hypixel.net, I am thrown an error as the connection times out. I have removed the optional timeout, but this issue persists. Thank you for any help you can provide!

**Comment by ldilley** [comment]
Greetings, @Archangel4148. Could you please share the programming language that you are using?

**Reply by Archangel4148** [reply]
Yes, sorry. I'm using Javascript. Looking at the documentation, I'm now realizing that the language may be my issue, looks like Javascript still only has minimal functionality.

**Reply by ldilley** [reply, marked as answer]
Unfortunately, that is likely the cause of your issue. Some servers like Hypixel ignore SLP requests below 1.7 (JSON). There is currently no estimated time of when the missing functionality might be added. In the meanwhile, you could try either of these JavaScript projects for your bot:
* https://github.com/Cryptkeeper/mcping-js
* https://github.com/minescope/mineping
